ee/tabby-db/src/users.rs

```diff
@@ -203,6 +206,23 @@ impl DbConn {
 
         Ok(())
     }
+
+    pub async fn update_user_active(&self, id: i32, active: bool) -> Result<()> {
+        let new_active = self
+            .conn
+            .call(move |c| {
+                c.execute("UPDATE users SET active=? WHERE id=?", (active, id))?;
```

Comment:
can this being something like `UPDATE users SET active=? WHRERE id=? and active=?, (active, id, !active)`? So you could decide whether to throw error based on number of rows being changed (returned by query)